src/server/game/Chat/Channels/Channel.h

```diff
@@ -269,19 +286,20 @@ class TC_GAME_API Channel
         }
 
         typedef std::map<ObjectGuid, PlayerInfo> PlayerContainer;
-        typedef GuidSet BannedContainer;
+        typedef GuidUnorderedSet BannedContainer;
```

Comment:
Since GuidSet was changed to unordered, maybe PlayerContainer can be changed to unorderd as well?
Not sure if it can be just swapped. Cant see any uses in this diff for the map's ordering.